### DISCUSSION TITLE
Add a secondary battery

### DISCUSSION THREAD

**Discussion by lumiror** [slipx06/sunsynk-power-flow-card#232] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Only one battery.

### Expected behaviour

Show two batteries .

### Mode

Both

### Context / Reason

In addition to displaying several strings, display two batteries, as I have two packs with two BMS and I use the data to set the parameters from the BMS and not from the inverter. That's why I want to ask if it would be possible to add a second battery even at the expense of some other parameter (I understand that there is little space), so that the user would choose what to display as a priority for him. I, for example I don't need grid, aux. Thank you.

**Comment by slipx06** [comment]
Hi. Its not planned at the moment. Will keep it open to see if there is any other interest in this feature

**Reply by Roving-Ronin** [reply]
2 inverters and 2 batteries, even if it means a 'wide' mode card ;-)

**Comment by R0DRIG0MEL0** [comment]
also i have 2 battery packs, i am interested

**Comment by Wrightsteve** [comment]
Het there. I have 2 inverters and 2 batteries too. May be more than 2 batteries going forward.
